po/de.po

```diff
@@ -207,156 +197,160 @@ msgstr ""
+#, c-format
 msgid "Delete %s"
-msgstr "Gelöscht "
+msgstr "Lösche %s"
 
 #: src/launchers/steam.vala:550
-#, fuzzy, c-format
+#, c-format
 msgid "Install %s"
-msgstr "Nur installierte"
+msgstr "Installiere %s"
 
 #: src/launchers/steam.vala:551
 msgid "Show more information"
-msgstr ""
+msgstr "Mehr Informationen anzeigen"
 
 #: src/launchers/steam.vala:558
 msgid "Check this to also delete your configuration files."
 msgstr ""
+"Aktivieren Sie diese Option, um auch Ihre Konfigurationsdateien zu löschen."
 
 #: src/launchers/steam.vala:560
 #, c-format
 msgid "You're about to delete %s from your system."
-msgstr ""
+msgstr "Sie sind dabei %s von Ihrem System zu löschen."
 
 #: src/launchers/steam.vala:560
-#, fuzzy
 msgid "Are you sure you want this?"
-msgstr "Sind Sie sicher, dass Sie löschen wollen "
+msgstr "Sind Sie sicher, dass Sie das wollen ?"
 
 #: src/launchers/steam.vala:562
 msgid "No"
-msgstr ""
+msgstr "Nein"
 
 #: src/launchers/steam.vala:563
 msgid "Yes"
-msgstr ""
+msgstr "Ja"
 
 #: src/launchers/steam.vala:579
-#, fuzzy, c-format
+#, c-format
 msgid "An unexpected error occurred while removing %s."
-msgstr "Ein unerwarteter Fehler trat auf beim Extrahieren von "
+msgstr "Ein unerwarteter Fehler trat auf beim Entfernen von %s."
 
 #: src/launchers/steam.vala:588
 #, c-format
 msgid "%s is already up-to-date."
-msgstr ""
+msgstr "%s ist auf dem aktuellen Stand."
 
 #: src/launchers/steam.vala:590
-#, fuzzy, c-format
+#, c-format
 msgid "The upgrade of %s has begun."
-msgstr "Die Löschung von "
+msgstr "Die Aktualisierung von %s hat begonnen."
 
 #: src/launchers/steam.vala:600
-#, fuzzy, c-format
+#, c-format
 msgid "The upgrade of %s is complete."
-msgstr "Die Löschung von "
+msgstr "Die Aktualisierung von %s ist abgeschlossen."
 
 #: src/launchers/steam.vala:602
-#, fuzzy, c-format
+#, c-format
 msgid "An unexpected error occurred while upgrading %s."
-msgstr "Ein unerwarteter Fehler trat auf beim Extrahieren von "
+msgstr "Ein unerwarteter Fehler trat auf beim Aktualisieren von %s."
 
 #: src/launchers/steam.vala:633 src/launchers/steam.vala:636
 #, c-format
 msgid "%s is not supported"
-msgstr ""
+msgstr "%s wird nicht unterstützt"
 
 #: src/launchers/steam.vala:633
 #, c-format
 msgid "To install %s for the %s, please run the following command:"
-msgstr ""
+msgstr "Um %s für %s zu installieren, bitte das folgende Kommando ausführen:"
 
 #: src/launchers/steam.vala:636
 #, c-format
 msgid "There's currently no known way for us to install %s for the %s."
 msgstr ""
+"Es gibt derzeit keine uns bekannte Möglichkeit, %s für die %s zu "
+"installieren."
 
 #: src/launchers/steam.vala:706
 #, c-format
 msgid "%s is up-to-date"
-msgstr ""
+msgstr "%s ist auf dem neuesten Stand"
 
 #: src/launchers/steam.vala:706
 #, c-format
 msgid "Update %s to the latest version"
-msgstr ""
+msgstr "%s auf die letzte Version aktualisieren"
 
 #: src/models/release.vala:63
-#, fuzzy
 msgid "Not Available"
 msgstr "Nicht verfügbar"
```

Comment:
`"Nicht Verfügbar"` (capitalization)

This is used for file size display in the UI when the size is unknown and is meant to be in Title Case.